efel/api.py

```diff
@@ -104,157 +89,138 @@ def reset():
     _initialise()
 
 
-def setDependencyFileLocation(location):
-    """Set the location of the Dependency file
-
-    The eFEL uses 'Dependency' files to let the user define which versions
-    of certain features are used to calculate.
-    The installation directory of the eFEL contains a default
-    'DependencyV5.txt' file. Unless the user wants to change this file,
-    it is not necessary to call this function.
+@deprecated("Changing the dependency file will not be supported in the future.")
+def setDependencyFileLocation(location: str | Path) -> None:
+    """Sets the location of the Dependency file.
 
-    Parameters
-    ==========
-    location : string
-               path to the location of a Dependency file
-    """
+    eFEL uses 'Dependency' files to let the user define versions of features to use.
+    The installation directory of the eFEL contains a default 'DependencyV5.txt' file.
+    Unless users want to change this file, it is not necessary to call this function.
 
-    global dependencyFileLocation
-    if not os.path.exists(location):
-        raise Exception(
-            "Path to dependency file {%s} doesn't exist" %
-            location)
-    _settings.dependencyfile_path = location
+    Args:
+        location: Path to the location of a Dependency file.
 
+    Raises:
+        FileNotFoundError: If the path to the dependency file doesn't exist.
+    """
+    location = Path(location)
+    if not location.exists():
+        raise FileNotFoundError(f"Path to dependency file {location} doesn't exist")
+    _settings.dependencyfile_path = str(location)
 
-def getDependencyFileLocation():
-    """Get the location of the Dependency file
 
-    The eFEL uses 'Dependency' files to let the user define which versions
-    of certain features are used to calculate.
-    The installation directory of the eFEL contains a default
-    'DependencyV5.txt' file.
+@deprecated("Changing the dependency file will not be supported in the future.")
+def getDependencyFileLocation() -> str:
+    """Gets the location of the Dependency file.
 
-    Returns
-    =======
-    location : string
-               path to the location of a Dependency file
+    Returns:
+        Path to the location of a Dependency file.
     """
-
     return _settings.dependencyfile_path
 
 
-def setThreshold(newThreshold):
+def set_threshold(new_threshold: float) -> None:
```

Comment:
why change the function name here?

Reply:
To make it Python style and also to make it consistent with the rest of Python modules.
https://pep8.org/#naming-conventions